src/main/java/core/basesyntax/Obtain.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Obtain {
```

Comment:
Bad naming, check [common mistakes file](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md) to have an example